app/models/concerns/commit_enrich.rb

```diff
@@ -0,0 +1,160 @@
+          end
+        end
+      end
+
+      if sort_opts.present?
+        sort_opts.each do |sort_opt|
+          if sort_opt.type == "commit_hash"
+            base = base.sort("hash" => sort_opt.direction)
+          else
+            base = base.sort(sort_opt.type => sort_opt.direction)
+          end
+        end
+      else
+        base = base.sort(sort => direction)
+      end
+
+      base
+
+    end
+
+    def fetch_commit_page_by_repo_urls(
+        repo_urls, begin_date, end_date, branch,
+        target: 'tag', filter: :grimoire_creation_date, sort: :grimoire_creation_date, direction: :desc,
+        per: 1, page: 1, filter_opts: [], sort_opts: []
+      )
+      base_commit_list_by_repo_urls(
+        repo_urls, begin_date, end_date, branch,
+        target: target, filter: filter, sort: sort, direction: direction,
+        filter_opts: filter_opts, sort_opts: sort_opts
+      )
+        .page(page)
+        .per(per)
+        .execute
+        .raw_response
+    end
+
+
+    def commit_count_by_repo_urls(
+      repo_urls, begin_date, end_date, branch,
+      target: 'tag', filter: :grimoire_creation_date, filter_opts: []
+    )
+
+      base_commit_list_by_repo_urls(
+          repo_urls, begin_date, end_date, branch,
+          target: target, filter: filter, filter_opts: filter_opts
+        )
+        .total_entries
+    end
+
+    def fetch_commit_contributor_list_by_repo_urls(repo_urls, begin_date, end_date, branch, target: 'tag', per: 20)
+      base_commit_list_by_repo_urls(repo_urls, begin_date, end_date, branch, target: target)
+        .aggregate(
+          "group_by_name": {
+            "terms": { "field": "author_email", "size": per },
+            "aggs": {
+              "lines_changed": { "sum": { "field": "lines_changed" } },
+              "lines_added": { "sum": { "field": "lines_added" } },
+              "lines_removed": { "sum": { "field": "lines_removed" } },
+              "author_domain": { "top_hits": { "_source": ["author_domain"], "size": 1 } },
+              "sort_bucket": { "bucket_sort": { "sort": [{ "lines_changed": { "order": "desc" } }] } }
+            }
+        })
+        .per(0)
+        .execute
+        .raw_response
+    end
+
+    def fetch_commit_organization_list_by_repo_urls(repo_urls, begin_date, end_date, branch, target: 'tag', per: 10000)
+       base_commit_list_by_repo_urls(repo_urls, begin_date, end_date, branch, target: target)
+        .aggregate(
+          "group_by_name": {
+            "terms": { "field": "author_domain", "size": per },
+            "aggs": {
+              "lines_changed": { "sum": { "field": "lines_changed" } },
+              "lines_added": { "sum": { "field": "lines_added" } },
+              "lines_removed": { "sum": { "field": "lines_removed" } },
+              "sort_bucket": { "bucket_sort": { "sort": [{ "lines_changed": { "order": "desc" } }] } }
+            }
+          })
+        .per(0)
+        .execute
```

Comment:
You can use the symlbol key of Hash directly, without adding `"` to the Hash key.